plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/BuildImageParams.java

```diff
@@ -40,7 +43,7 @@ public BuildImageParams withRepository(String repository) {
 
     /**
      * @param authConfigs
-     *         authentication configuration for private registries. Can be null
+     *         authentication configuration for registries. Can be null
```

Comment:
Or as parameters of docker mostly optional we can consider all parameters nullable and add required annotation to those that are not. If so we need package docs.